src/System.Globalization.Extensions/tests/Normalization/StringNormalizationTests.cs

```diff
@@ -32,13 +32,8 @@ public void IsNormalized_Invalid()
             Assert.Throws<ArgumentException>("strInput", () => "\uD800\uD800".IsNormalized()); // Invalid surrogate pair
 
             Assert.Throws<ArgumentNullException>("strInput", () => StringNormalizationExtensions.IsNormalized(null));
-            
-            Exception exception = Record.Exception(() => ((string)null).IsNormalized());
-            
-            // On desktop IsNormalized is not extension method, trying to do ((string)null).IsNormalized()
-            // will get NullReferenceException, in .Net Core we use extension method which will throw
-            // ArgumentNullException
-            Assert.True((exception is ArgumentNullException) || (exception is NullReferenceException));
+            // Ensure that extension method not used by current framework.
+            Assert.Throws<NullReferenceException>(() => ((string)null).IsNormalized());
```

Comment:
I think we can drop this assert entirely: calling an instance method on a null instance produces a NullReferenceException... that's a runtime feature, not something specific to this lib.